### ISSUE TITLE
List: ariaProvider and accessibilityProvider should be merged

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#94544]
List `ariaProvider` and `accessibilityProvider` should be merged into one which has the mendatory `getAriaLabel`.
Code pointer https://github.com/microsoft/vscode/blob/404f66c13cb3c4ef0fb30db8a09b2967fc4b8367/src/vs/base/browser/ui/list/listWidget.ts#L846

**Comment by joaomoreno** [comment]
With this I've also fixed quick open, suggest widget, etc, which were all returning an off-by-one `aria-posinset`.

**Comment by isidorn** [comment]
Thanks, thus you have also fixed this one https://github.com/microsoft/vscode/issues/93924
